packages/cadl-typescript/test/integration/generated/array/itemTypes/cadl-project.yaml

```diff
@@ -0,0 +1,12 @@
+emitters:
+  "../../../../../../../../../../../packages/cadl-typescript/dist/src/index.js":
```

Comment:
I wonder if we can just have @azure-tools/cadl-typescript and rush resolve it to the local project?

Reply:
Actually we have tried before, As this path is resolved by cadl compiler which doesn't belong to this project, rush can not help us.